### PR TITLE
Use proto3

### PR DESCRIPTION
**Pull request by voulg**
Use proto3

### CODE REVIEW

File: server/src/main/proto/service.proto [hunk 1]
```diff
@@ -42,85 +42,85 @@ service FinApp {
...
+  string new_balance = 1; // Digits[.[Digits]] or [Digits].Digits
 }
 
 message TransactionEntry {
-  optional bytes account_id = 1;
-  optional .google.protobuf.Timestamp event_timestamp = 2;
-  optional bool is_credit = 3;
-  optional string amount = 4;
+  bytes account_id = 1;
+  .google.protobuf.Timestamp event_timestamp = 2;
```
Comment (c24t) [inline server/src/main/proto/service.proto:112]: These can just be `Timestamp`s now right? https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp

**Review by c24t — APPROVED** [review]
LGTM after addressing the timestamp type.

File: server/src/main/proto/service.proto (hunk 1 above)
Comment (voulg) [reply server/src/main/proto/service.proto:112]: I do not think so. The code you link is java not proto and the page says that these definitions are in the package google.protobuf . I also tried it and did not work... Let me know if there are reasons you expect it to work without the package name.

Comment (c24t) [reply server/src/main/proto/service.proto:112]: Nevermind, you're right.

**Pull request merged**